### PR TITLE
[13.0][ADD] account_financial_report_menu

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
Change the menu position "Invoicing/Reports/OCA accounting reports" to "Invoicing/OCA accounting reports"

### CODE REVIEW

**Comment by ChristianSantamaria** [comment]
@dalonsod how do you see it?

**Comment by dalonsod** [comment]
:+1: could you install it in AluTest environment?

**Comment by ChristianSantamaria** [comment]
Done :+1:

File: account_financial_report_menu/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "summary": """
+        Change the menu position "Invoicing/Reports/OCA accounting reports" to "Invoicing/OCA accounting reports"
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "13.0.1.0.0",
+    "category": "Account",
+    "website": "https://github.com/solvosci/slv-account",
+    "depends": [
+        "account_financial_report",
```
Comment (dalonsod) [inline account_financial_report_menu/__manifest__.py:14]: Please add to `oca_dependencies.txt` a reference to `account-financial-reporting`.

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comments.
And also, could you try unistalling addon e.g. in your Dev environment and check if menu is restored to original place? If not, could you try an uninstall hook in order to get back original parent menu? See https://github.com/OCA/sale-workflow/blob/4227c9f2b9e4fbde8b2e3f2867cf0756cf248394/portal_sale_personal_data_only/hooks.py#L15

**Comment by ChristianSantamaria** [comment]
Done :+1:

**Comment by ChristianSantamaria** [comment]
Miss oca_dependencies.. now its done

**Review by dalonsod — APPROVED** [review]
LGTM,

We'll actually install it as a beta development, because we are not 100% secure about read permissions of every Invoice user for every report (only General Ledger is validated).

**Pull request merged**